openshift_scalability/scripts/ptibenchmark.sh

```diff
@@ -0,0 +1,20 @@
+# The following script will execture A B Benchmark for pti(spectre).
+
+#Usage:
+#	nohup ./ptibenchmark.sh 2>&1 &
+
+#TODO: Abstract this script to have an A B node vertical concept, but do some changes in between the runs.
+
+#Optimize the register script to run this command there.
+for h in `oc get no |grep -v NAME|awk '{print $1}'`; do ssh $h "echo "export LC_CTYPE=en_US.UTF-8 >> ~/.bashrc; export LC_ALL=en_US.UTF-8 >> ~/.bashrc; pbench-register-tool-set; done
+
+#PTI Enabled
+for h in `oc get no |grep -v NAME|awk '{print $1}'`; do ssh $h "echo 1 > /sys/kernel/debug/x86/pti_enabled"; done 
+pbench-user-benchmark ~/svt/openshift_scalability/./nodeVertical.sh ptibench python > /tmp/n.log
+pbench-move-results --prefix=pti_enabled
+
+#PTI Disabled
+for h in `oc get no |grep -v NAME|awk '{print $1}'`; do ssh $h "echo 0 > /sys/kernel/debug/x86/pti_disabled"; done 
```

Comment:
To disable the patch:   echo 0 > /sys/kernel/debug/x86/pti_enabled   See:  https://access.redhat.com/articles/3311301

Reply:
seems like to completely disable it we need 3 of them to be 0 pti, ibpb, ibrs
will change that